src/main/java/org/json/XMLtoJSONMLParserConfiguration.java

```diff
@@ -0,0 +1,128 @@
+package org.json;
+/*
+Public Domain.
+*/
+
+/**
+ * Configuration object for the XML to JSONML parser. The configuration is immutable.
+ */
+@SuppressWarnings({""})
+public class XMLtoJSONMLParserConfiguration {
```

Comment:
Is there a reason for not re-using XMLParserConfiguration?
Either way, please don't expose any new APIs for JSONML besides the max nesting depth.

Reply:
The main reason: not all options of XMLParserConfiguration is supported with JSONML parsing, so it seemed cleaner to create a new config class.

Reply:
The new APIs for JSONML:
- public static JSONObject toJSONObject(String string, XMLtoJSONMLParserConfiguration config)
- public static JSONObject toJSONObject(XMLTokener x, XMLtoJSONMLParserConfiguration config)

These are both necessary to support max nesting depth and the `keepStrings` option in one function.
The alternative would be to add create 4 new `toJSONObject` functions that include the `int maxNestingDepth` parameter, on top of the existing 4 such functions - the potential parameter combinations doubling with a new optional parameter.

Is this acceptable?

Reply:
Just noticed that I did not add the max depth limit override option for the `toJSONArray` calls - await a fix soon.